packages/actor-tests/lib/faucet.ts

```diff
@@ -0,0 +1,27 @@
+import { ethers, providers } from 'ethers'
+import { fetchJson, getAddress } from 'ethers/lib/utils'
+
+export class Faucet {
+  private url: string
+
+  private provider: providers.Provider
+
+  constructor(url: string, provider: providers.Provider) {
+    this.url = url
+    this.provider = provider
+  }
```

Comment:
Don't change this but in case you didn't know, you can write this with way less boilerplate

```typescript
constructor(
  private url: string,
  private provider: providers.Provider,
) {}
```